mottak/src/main/kotlin/no/nav/melosys/soknadmottak/soknad/SoknadCache.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.melosys.soknadmottak.soknad
+
+import mu.KotlinLogging
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+@Component
+class SoknadCache @Autowired constructor(
```

Comment:
SoeknadStatCache, SoknadStatistikkCache eller tilsvarende?
Det er ikke en cache for søknader